src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -169,6 +177,35 @@ func useAllCores() {
 	runtime.GOMAXPROCS(numcores)
 }
 
+func configureLogging() *rotator.SizeRotator {
+	logdir := appdir.Logs("Lantern")
+	log.Debugf("Placing logs in %v", logdir)
+	if _, err := os.Stat(logdir); err != nil {
+		if os.IsNotExist(err) {
```

Comment:
>  -rw-r--r--  1 root    staff  2032 Jan 15 19:47 installer.log
>  -rw-r--r--  1 root    staff     0 Jan 15 19:47 install4jout.txt
>  -rw-r--r--  1 root    staff     0 Jan 15 19:47 install4jerr.txt

It was created during installation of Lantern java version, so it's vital to ensure writability. We may need to fix it in installer, or just remove and recreate whole directory if not writable.